test/js/css-token-stream-value.js

```diff
@@ -0,0 +1,69 @@
+suite('CSSTokenStreamValue', function() {
+  test("CSSTokenStreamValue is a CSSTokenStreamValue and CSSStyleValue", function() {
+    assert.instanceOf(new CSSTokenStreamValue(), CSSTokenStreamValue,
+      'A new CSSTokenStreamValue should be an instance of CSSTokenStreamValue');
```

Comment:
These messages are unnecessary - the default error message for assert.instanceOf is sufficient.